src/acb_vector.jl

```diff
@@ -0,0 +1,35 @@
+@inline clear!(acb_vs::acb_vec_struct) = clear!(acb_vs.entries, acb_vs.n)
+Base.size(acb_v::acb_vec_struct) = (acb_v.n,)
+Base.getindex(v::acb_vec_struct, i::Integer) = v.entries + (i - 1) * sizeof(acb_struct)
```

Comment:
Should we add boundary checks here? These are internal functions so it might not be as important. If we do add it we should add explicit `@inbounds` to the internal usages of this.